packages/ecc-client-ga4gh-tes/src/components/tes-run/definition/tesRun.template.ts

```diff
@@ -42,6 +42,20 @@ provideFASTDesignSystem().register(
   })
 );
 
+const state = [
```

Comment:
Make sure to initialize the state variable only once across the files!

Reply:
Yeah :+1: , I did, did you find somewhere it defined twice :sweat_smile: ?, it is but not in this package and hence I can't import from there.

Reply:
In file `packages/ecc-client-ga4gh-tes/src/components/tes-runs/definition/tesRuns.ts` line:11.

Reply:
:man_facepalming: Yeah! I totally missed it. done!